docs/tutorials/mastering.md

```diff
@@ -1117,84 +1347,147 @@ We will now add a subscriber with the IMSI that was provided to the gNB simulato
 - Network Slice: `Tutorial`
 - Device Group: `Tutorial-default`
 
-## 8. Integrate SD-Core with Observability
+## 8. Integrate SD-Core with the Canonical Observability Stack (COS)
 
-We will integrate the 5G core network with the Canonical Observability Stack (COS). All commands are to be executed on the `juju-controller` VM.
+The following steps show how to integrate the SD-Core 5G core network with the Canonical Observability Stack (COS).
 
-### Deploy the `cos-lite` bundle
+First, we will add COS to the Terraform module used in the previous steps. 
+Next, we will expose the Software as a Service offers for the COS and create integrations with SD-Core 5G core network components.
 
-Create a new Juju model named `cos` on the `juju-controller` cloud:
+### Deploy COS Lite
+
+Add `cos-lite` Terraform module to the `main.tf` file used in the previous steps:
```

Comment:
We are missing the `cd terraform` and ensuring the user is logged into the controller VM

Reply:
Do we need that? If the user follows the tutorial, both will be ensured anyway.

Reply:
true. It just felt inconsistent with other places.